### PR TITLE
Add alternative English names for China, Gambia, and Tanzania.

### PR DESCRIPTION
**Pull request by iamleeg**
In the case of the Republic of The Gambia, that's its official name, but Mapbox use "The Gambia" so it's useful to have both. "Tanzania" is a common synonym for the United Republic of Tanzania; again Mapbox uses this in its geocoding API. My motivation is that I'm trying to transform country names returned from the Mapbox API into ISO codes, hence adding the names used over there.

### CODE REVIEW

File: langs/en.json [hunk 1]
```diff
@@ -208,7 +208,7 @@
     "SY": "Syrian Arab Republic",
     "TW": ["Taiwan, Province of China", "Taiwan"],
     "TJ": "Tajikistan",
-    "TZ": "Tanzania, United Republic of",
+    "TZ": ["Tanzania, United Republic of", "Tanzania"],
```
Comment (michaelwittig) [inline langs/en.json:211]: While you are at it, could you change Tanzania, United Republic of to United Republic of Tanzania as well?

**Comment by iamleeg** [comment]
@michaelwittig what do you think of adding former names? E.g. FYROM for North Macedonia, Swaziland for Eswatini. These are cases that also got named that way by Mapbox (after the name changes were enacted) but I am willing to special-case them if you don't want that legacy baggage in this library.

**Comment by michaelwittig** [comment]
Sorry for my late reply. I want to check the official name used by ISO but their site is down since Wednesday... https://www.iso.org/sites/outage/#iso:code:3166:CN

**Comment by michaelwittig** [comment]
@iamleeg The library has now way to support historical names today. I suggest to special-case that in your code as you suggested.

**Pull request merged**